test.sh

```diff
@@ -15,9 +14,9 @@ run_with_suppressed_output () {
 source ./versions.sh
 
 if [ -n "${CI_VERSION:-}" ]; then
-	meteor_versions=( "$CI_VERSION" )
+	meteor_versions=("$CI_VERSION")
 elif [[ "${1-x}" != x ]]; then
-	meteor_versions=( "$1" )
+	meteor_versions=("$1")
```

Comment:
I assume these whitespace changes are unnecessary for what you’re trying to achieve? Can you please undo all the whitespace changes (here, and elsewhere in this file) so that this PR is just what’s necessary? I’d like to keep the coding style consistent across all the shell scripts in the repo.

Reply:
Yeah sorry habit. Reverted.